### ISSUE TITLE
Средний заработок при расчете отпусков.

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#3153]
Originally reported on Google Code with ID 3153

```
В док-те "Начисление отпусков" при расчете среднего заработка, если у работника был
перевод (приказ о переводе) в середине месяца, то этот месяц попадет двумя строчками
(первая строчка это начало месяца, вторая - дата перевода) и ЗП в каждую строчку попадет
полностью за весь месяц.
```

Reported by `spiker2005.moved@tut.by` on 2013-06-11 14:40:10

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2013-06-11 15:40:36
- **Labels added**: Type-Defect, Priority-Medium, Wage

**Comment by gsbelarus** [comment]
Reported by `roberto1813` on 2013-06-11 17:08:09
- **Status changed**: `Duplicate`
- **Merged into**: #3038
